server/data/configClient.js

```diff
@@ -0,0 +1,66 @@
+const db = require('./dataAccess/db')
+
+module.exports = {
+  async getAllMailboxes() {
+    const query = {
+      text: 'select * from notifications_config order by establishment ASC',
+    }
```

Comment:
Would it be better to select specific columns?  Any change in table structure would also change the object produced from these queries. Its fine if that is the intention though.

Reply:
agreed